### PR TITLE
Fix failed tests in georchestra/gateway

### PR DESCRIPTION
**Pull request by marwanehcine**
This pr contains changes required to fix failed test for AccessRulesCustomizerIT, georchestra/gateway

### CODE REVIEW

**Comment by jeanmi151** [comment]
This should not be merged into the branch master because it contains docker-master value !
I switch the branch to merge into to docker-master

File: gateway/gateway.yaml [hunk 1]
```diff
@@ -106,6 +106,8 @@ georchestra:
       header: 
         target: http://header:8080/header/
         access-rules:
+        - intercept-url: /header/ogcproxy/**
```
Comment (fvanderbiest) [inline gateway/gateway.yaml:109]: there's no ogcproxy in /header to my knowledge

**Comment by fvanderbiest** [comment]
@marwanehcine can you double check this one ?
Thanks

File: gateway/gateway.yaml (hunk 1 above)
Comment (marwanehcine) [reply gateway/gateway.yaml:109]: ça existe uniquement dans security-proxy (bean spMetadataSource de type securityMetadataSource).
Dans le header path ogcproxy/ n'existe plus aprés un code check rapide!

Comment (emmdurin) [reply gateway/gateway.yaml:109]: This change seems to have no effect.

Comment (fvanderbiest) [reply gateway/gateway.yaml:109]: Please remove

**Comment by fvanderbiest** [comment]
Conflict to be resolved

**Pull request closed without merging**